### PR TITLE
Move Grafana Log Display to a tab

### LINKED ISSUE #27: Move Grafana Log Into a Tab

### PR DESCRIPTION
**Pull request by cristinasewell**
This needs to be tested again to make sure I did not change anything that I was not supposed to....  but this is kind of how it looks now:

Please let me know if I should change anything about this. 

last

We could expend the `Requested Beam Parameters` and the `ev`  - but I am not sure if it will look better, maybe in the empty part on top of the screen now we can add more info?

Trying to close #27

### CODE REVIEW

**Comment by ZLLentz** [comment]
Wow, it's really hard to parse a ui file diff. I'm going to open this, play around with it, and compare with master to see if I notice anything strange.

**Comment by cristinasewell** [comment]
Ohh I got it now, I should start tagging people in `pmps-ui` - my bad, I thought people are watching this repo (I should never assume that)

Thanks for looking at this repo @ZLLentz

**Comment by ZLLentz** [comment]
I think it's our bad for not watching too- I'm surprised it was literally nobody!

**Review by ZLLentz — APPROVED** [review]
I think this one looks really good. Two things I noticed that can easily be dropped or done in a follow-up PR:
- The eV box seems to really want to stretch itself as you expand the window horizontally. I think it might just need a maximum horizontal width parameter?
- The embedded grafana table has some really strange scaling where it doesn't want to take up the whole tab. I think this must be on the server side though? I may investigate later.

I'm pretty happy to just merge this as-is

**Comment by cristinasewell** [comment]
> * The eV box seems to really want to stretch itself as you expand the window horizontally. I think it might just need a maximum horizontal width parameter?

Ohh thanks for noticing that, I will add a maximum horizontal width. 

> * The embedded grafana table has some really strange scaling where it doesn't want to take up the whole tab. I think this must be on the server side though? I may investigate later.

I think it is on the server side because I've seen it on the main screen too, but I could be wrong.

**Comment by cristinasewell** [comment]
Ok I set the horizontal sizePolicy for the eV frame to `maximum` and now it should just take as much as needed instead of expending (i think) - will look similar to what we had before, it leaves more free space on the right of the window but we could later add something else there
grafana_v2

**Comment by ZLLentz** [comment]
I think that looks perfect! I'm going to merge because I like it a lot (and because it may interfere with other PRs and we'd rather know sooner than later)

The grafana thing must 100% be server-side. I think we figure out how to adjust the way the server shows that window once we've deployed the screen with the grafana log tab. For now we can't risk breaking the live display (and even though this looks sort of odd here, it doesn't stop us from using the ui)
